### PR TITLE
translate docs/zh/operations/table_engines/aggregatingmergetree.md

### PR DESCRIPTION
**Pull request by neverlee**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

For changelog. Remove if this is non-significant change.

Category (leave one):
- Other

Short description (up to few sentences):
translate docs/zh/operations/table_engines/aggregatingmergetree.md
...

Detailed description (optional):

...

### CODE REVIEW

**Comment by blinkov** [comment]
cc @zhang2014 @sundy-li

File: docs/zh/operations/table_engines/aggregatingmergetree.md [hunk 1]
```diff
@@ -44,19 +44,19 @@ CREATE TABLE [IF NOT EXISTS] [db.]table_name [ON CLUSTER cluster]
...
-All of the parameters have the same meaning as in `MergeTree`.
+上面的所有参数跟 `MergeTree` 中的一样。
 </details>
 
-## SELECT and INSERT
+## SELECT 和 INSERT
 
-To insert data, use [INSERT SELECT](../../query_language/insert_into.md) query with aggregate -State- functions..
-When selecting data from `AggregatingMergeTree` table, use `GROUP BY` clause and the same aggregate functions as when inserting data, but using `-Merge` suffix.
+插入数据，需使用带有统计 -State- 函数的 [INSERT SELECT](../../query_language/insert_into.md) 语句。
```
Comment (zhang2014) [inline docs/zh/operations/table_engines/aggregatingmergetree.md]: 统计函数 -> 聚合函数？

**Comment by zhang2014** [comment]
LGTM

File: docs/zh/operations/table_engines/aggregatingmergetree.md (hunk 1 above)
Comment (neverlee) [reply docs/zh/operations/table_engines/aggregatingmergetree.md]: 确实这么看起来合适些。改了

**Pull request merged**